### ISSUE TITLE
[develop] Hotkeys broken

### ISSUE THREAD

**Issue by asergi** [nicolargo/glances#967]
- `'3'` does nothing
- `'5'` doesn't hide top menu
- if you hit the `'g'` key Glances crashes:
```
Traceback (most recent call last):
  File "/home/alessio/venvs/py35-glances/bin/glances", line 9, in <module>
    load_entry_point('Glances==2.7.1', 'console_scripts', 'glances')()
  File "/home/alessio/venvs/py35-glances/src/glances/glances/__init__.py", line 229, in main
    start_standalone(config=config, args=args)
  File "/home/alessio/venvs/py35-glances/src/glances/glances/__init__.py", line 112, in start_standalone
    standalone.serve_forever()
  File "/home/alessio/venvs/py35-glances/src/glances/glances/standalone.py", line 111, in serve_forever
    return self.__serve_forever()
  File "/home/alessio/venvs/py35-glances/src/glances/glances/standalone.py", line 96, in __serve_forever
    self.screen.update(self.stats)
  File "/home/alessio/venvs/py35-glances/src/glances/glances/outputs/glances_curses.py", line 978, in update
    pressedkey = self.__catch_key(return_to_browser=return_to_browser)
  File "/home/alessio/venvs/py35-glances/src/glances/glances/outputs/glances_curses.py", line 334, in __catch_key
    self._hotkeys[hotkey]['switch']))
AttributeError: 'Namespace' object has no attribute 'graph_tag'
```

**Comment by nicolargo** [comment]
Corrected !

Thanks for the head up @asergi

**Comment by asergi** [comment]
After the recent refactoring, the following hotkeys don't work: `4`, `5`, `r`, `s`, `d`, `f`, `l`, `n`.

**Comment by nicolargo** [comment]
Work on it...

**Comment by nicolargo** [comment]
Hotkeys corrected in the latest commit.

Thanks for the headup and good new year's eve !

**Comment by asergi** [comment]
Happy New Year's Eve you too!

**Comment by asergi** [comment]
3rd (final?) round: `l` (again), `D`, `I`.

**Comment by nicolargo** [comment]
Corrected in the last commit 9c0e27c

Thanks for the head up @asergi  !
